stack_master/config/DEFAULT/kinematic_mpc_params.yaml

```
@@ -0,0 +1,48 @@
+delta_max: !!float 0.40  # maximum steering angle [rad]
+# Velocity constraint
+v_min: !!float 2 # minimum velocity [m/s]
+v_max: !!float 12 # maximum velocity [m/s]
+
+# input bounds
+ddelta_min: !!float -0.2 # minimum change rate of stering angle [rad/s]
+ddelta_max: !!float 0.2 # maximum change rate of steering angle [rad/s]
+a_min: !!float -10  # minimum velocity change rate
+a_max: !!float 10  # maximum velocity change rate
+
+v_min: !!float 2 # minimum velocity [m/s]
```

YAMLlint (1.35.1)

[error] duplication of key "v_min" in mapping

(key-duplicates)